common/components/query/player.ts

```diff
@@ -17,3 +17,9 @@ export function hasStatusEffect(
 		return !!player.hasStatusEffect(statusEffect)
 	}
 }
+
+export function id(id: string): ComponentQuery<PlayerComponent> {
+	return (_game, player) => {
+		return player.id === id
+	}
+}
```

Comment:
I would prefer if you did not write a combinator for this.
Normally it would be okay, but I am trying to decouple player models from their IDs. Merging this than undoing it would end up being a bit of a waste of time so I would prefer if you used my alternate suggestion for the api.